### ISSUE TITLE
Kitchen timers and alarm clocks

### ISSUE THREAD

**Issue by Psiclops** [CleverRaven/Cataclysm-DDA#49029]
Kitchen timers and alarm clocks should be usable as a timed "sound bomb". I know alarm clocks already exist, maybe add egg timers, kitchen timers, and the like as common household items.  They could be electrical using ultra-light battery, or mechanical like the old school twist for 30 minutes style kitchen timer. Or both. 

With higher levels of electronics/fabrication skill, they should be attachable to other items to turn them on/detonate them. Example: kitchen timer attached to compressed air horn or stereo system or various explosives.

Basically a way to leave a delayed distraction device as a sneaky sneaky method of clearing a path.

It would be fantastic if the following scenario were also possible. Timer for 1 minute on a compressed air horn inside the house, sitting next to a 3 minute timer on an incendiary grenade or dynamite.

**Comment by DalenTalas** [comment]
I would suggest moving the "set alarm" option from the rest menu to the [a]ction on the watch / clock / timer.

As an additional idea, include grandfather clocks as furniture in the game, which would make a *lot* of noise every hour on the hour.

**Comment by Zireael07** [comment]
Related: #50414
